src/analysis/slice/data-flow.ts

```diff
@@ -337,6 +337,11 @@ export class DataflowAnalyzer {
         uses = undefinedRefs.filter(r => r.level == ReferenceType.USE);
         break;
       case ast.CLASS:
+        // For each class function, call getUses
```

Comment:
nit-pick on comment: won't this call getUses for every statement in the function (including class variable defs?)

Reply:
Updated the comment!